versions/models.py

```diff
@@ -831,8 +831,12 @@ def restore(self, **kwargs):
                 # Set all non-provided ForeignKeys to None.  If required, raise an error.
                 try:
                     setattr(restored, field.name, None)
-                except ValueError as e:
-                    raise ForeignKeyRequiresValueError(e.args[0])
+                    # Check for non null foreign key removed in Django 1.11
+                    # https://docs.djangoproject.com/en/1.11/releases/1.10/#removed-null-assignment-check-for-non-null-foreign-key-fields
```

Comment:
Thanks for this one. Currently working on a proposal to port CleanerVersion to Django 1.10 and your hint came in handy.